Add November 2022 newsletter (#1838)
writethedocs/www: docs/blog/newsletter-november-2022.rst

.. post:: November 07, 2022
   :tags: newsletter

#########################################
Write the Docs Newsletter – November 2022
#########################################

> Hello once again from the newsletter team. We have a lot of things going on in the community right now.

Reviewer: This we team / we community feels confusing to me. How about 

```suggestion
Hello once again from the newsletter team. There are many things going on in the community right now.
```

> I wrote last month about opportunities to be a part of the Write the Docs community, and this month sees more information. A `new Write the Docs Enhancement Proposal <https://github.com/writethedocs/weps/pull/6>`__ has been opened to provide more clarity on Write the Docs teams and what they do. The change also includes Mikey Ariel stepping down as Prague conference chair and starting as Community Board chair. While I'm sad to see an end to an era of conferences very near me, I'm excited for what she'll bring in the newly defined role. In related news, we have the dates for `conferences in 2023 </blog/2023-conference-announcements/>`__ that you'll probably want to save.

Reviewer: Strictly speaking, this is about being more involved, not about being part of the community (arguably by reading you're already part, what do you think?

Reviewer: Also arguably, it's not an end to the Prague conferences, just an announcement that one is not happening in 2023.

Author: Good points, both of them.

> Of course, 2022 isn't over yet and there's still the exciting `Write the Docs Australia conference </conf/australia/2022/>`__ to come. The `full schedule </conf/australia/2022/news/announcing-schedule/>`__ has just been announced, so check out all the interesting talks. There will also be workshops led by people from `The Good Docs Project <https://thegooddocsproject.dev/>`__. Be sure to `get your tickets soon </conf/australia/2022/tickets/>`__ as ticket sales close at the end of the month.

Reviewer: <3

As you're evaluating the context, consider where you need to form relationships around the organization to help the team work effectively. Find out what business goals the team helps with and what needs other teams have that the documentation team could help with. Learning this context helps you determine how you can best advocate for your team.

As you learn how to advocate for your team, think about how to grow it to meet the needs you discovered. This can mean helping people with professional development or figuring out who can be hired and how to get the budget for that. Creating a long-term plan based on what you determine in the first few months can help you place your team needs in a context that will help get them addressed.

----------------------------
Minimum viable documentation
----------------------------

> Minimum viable documentation is the minimum required information that's usable and helpful to users. For many reasons, it’s seen positively by some and adversely by others. Those who support it say it’s the best place to start if a product lacks any documentation. Additionally, in places where releases are frequent, minimal documentation enable you to write the essential information and then return to the document to update it once things have stabilized. But there are competing views on minimal documentation, and those who work at places without it believe that the "document everything" approach can lead to many issues, including leaning more towards a mindset of "done is better than perfect."

Reviewer 2: ```suggestion
Minimum viable documentation is the minimum required information that's usable and helpful to users. For many reasons, it’s seen positively by some and adversely by others. Those who support it say it’s the best place to start if a product lacks any documentation. Additionally, in places where releases are frequent, minimal documentation enables you to write the essential information and then return to the document to update it once things have stabilized. But there are competing views on minimal documentation, and those who work at places without it believe that the "document everything" approach can lead to many issues, including leaning more towards a mindset of "done is better than perfect."
```

Proponents of minimal documentation consider that a product should be capable of functioning with little explanation from its authors. They say this provides a quicker turnaround time on new features, enables faster bug fixes, and simplifies developers’ lives by removing tiresome writing duties. Opponents claim that this strategy results in varying quality across products and can lead to knowledge gaps within teams. They argue that all software should be documented since it eliminates uncertainty when using new systems (and makes life easier for users).

> If you’re interested in learning more about minimal viable documentation, read this `blog post from Sarah Moir <https://thisisimportant.net/posts/from-nothing-to-something-with-minimum-viable-documentation/>`__. It covers defining such documentation, assessing documentation states, and structuring documentation.

Reviewer 2: I noticed this item switches between `minimum` and `minimal` -- I'm not sure if there's a distinction made in the original conversation but it seems like they're being used interchangeably here.

Author: That's probably just my editing mistake. Thanks for the note!
